packages/react-components/src/atoms/MultiSelect.tsx

```diff
@@ -148,6 +149,7 @@ const MultiSelect = <T extends MultiSelectOptionsType>({
   components,
   enabled = true,
   placeholder = '',
+  maxMenuHeight,
```

Comment:
Shouldn't be added in here a default value, since this prop is optional?

Reply:
No, as you said it is optional. If it's not set I don't want to use it.